modules/cas_cache/layer_cache_management.c

```diff
@@ -148,15 +148,28 @@ static int _cache_mngt_async_caller_set_result(
 	return result;
 }
 
-static inline void _cache_mngt_async_context_init(
+static inline void _cache_mngt_async_context_common(
```

Comment:
I'd call it `_cache_mngt_async_context_init_common()`, because right now it's not very descriptive.